### PR TITLE
remove unused else from the Observable

### PR DESCRIPTION
**Pull request by VovaStelmashchuk**
Remove unused else from the Observable class.

### CODE REVIEW

File: src/main/java/io/reactivex/Observable.java [hunk 1]
```diff
@@ -13,26 +13,70 @@
 
 package io.reactivex;
 
-import java.util.*;
-import java.util.concurrent.*;
-
 import org.reactivestreams.Publisher;
 
-import io.reactivex.annotations.*;
+import java.util.Collection;
```
Comment (akarnokd) [inline src/main/java/io/reactivex/Observable.java]: Please do not unroll star imports.

**Review by akarnokd — CHANGES_REQUESTED** [review]

File: src/main/java/io/reactivex/Observable.java (hunk 1 above)
Comment (VovaStelmashchuk) [reply src/main/java/io/reactivex/Observable.java]: Fixed.

**Review by akarnokd — APPROVED** [review]

**Review by vanniktech — APPROVED** [review]

File: src/main/java/io/reactivex/Observable.java (hunk 1 above)
Comment (VovaStelmashchuk) [reply src/main/java/io/reactivex/Observable.java]: @akarnokd Maybe you can include some rule for the import bloc to `CONTRIBUTING.md` ?

Comment (akarnokd) [reply src/main/java/io/reactivex/Observable.java]: @jordan1997 

> When submitting code, please make every effort to follow existing conventions and style in order to keep the code as readable as possible.

But don't worry, we will warn you if there is a stylistic problem.

**Pull request merged**